contributivity_measures.py

```diff
@@ -138,3 +136,288 @@ def not_twice_characteristic(permut):
 
     return({'sv':sv, 'std_sv': np.std(contributions,axis=0) / np.sqrt(t-1),'prop':sv/np.sum(sv), 'computed_val':char_value_dict})
 
+
+#%% compute Shapley values with the linear importance sampling method
```

Comment:
Could you add a brief description of this method or a link to a paper ?

Reply:
The IS method is a first applicating of the importance sampling method. See section 9.1 in [this course](https://statweb.stanford.edu/~owen/mc/Ch-var-is.pdf). I try too approximate the optimal density by doing a linear interpolation of the   contributions: If V is the characteristic function, I compute the first contribution  `A=V(D_k)-V( empty_set )` and the last  one `B=V(I)- V(I\ D_k)`, then I approximated any other contribution `V(S U D_k)- V(S)` by `A*|S|/|I\D_K| +B*(1-|S|/|I\D_K| )`. Then I plug this approximation in the formula of the optimal density of importance sampling, in oder to get my importance density.